libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/ServiceCollectionExtensions.cs

```diff
@@ -57,7 +57,14 @@ public static IServiceCollection AddBotApplicationInsights(this IServiceCollecti
             IBotTelemetryClient telemetryClient = null;
             if (appInsightService != null)
             {
-                services.AddApplicationInsightsTelemetry(appInsightService.InstrumentationKey);
+                // Create Application Insights options and disable adaptive sampling
+                var aiOptions = new Microsoft.ApplicationInsights.AspNetCore.Extensions.ApplicationInsightsServiceOptions
```

Comment:
Can we simplify this by adding a using for `Microsoft.ApplicationInsights.AspNetCore.Extensions` ?

Reply:
you're right. Fixed.